extensions/android-paging3/src/test/java/com/squareup/sqldelight/android/paging3/OffsetQueryPagingSourceTest.kt

```diff
@@ -40,39 +38,28 @@ import kotlin.test.assertFailsWith
 @RunWith(RobolectricTestRunner::class)
 class OffsetQueryPagingSourceTest {
 
-  private lateinit var driver: SqlDriver
-  private lateinit var transacter: Transacter
-
-  @Before fun before() {
-    driver = AndroidSqliteDriver(
-      object : Schema {
-        override val version: Int = 1
-
-        override fun create(db: SqlDriver) {
-          db.execute(null, "CREATE TABLE testTable (value INTEGER PRIMARY KEY)", 0)
+  @Test fun `empty page gives correct prevKey and nextKey`() {
```

Comment:
This was the only testcase that I added. The rest of the changes were some refactoring such that the driver could be reused with a variable `rowCount`.

Reply:
Rather than refactoring all of the tests, would it be easier to just `DELETE FROM value` before this test code is executed?

Reply:
Yeah I was thinking about that but I felt that the test would read kinda weird, like if you read the test in isolation, it'd seem rather odd that the first thing that's being done is emptying out the table. You'd have to go checkout the `driver` implementation and see that the table is being populated with seemingly random data (if we hadn't familiarised ourself with the other tests).

That was my thought process behind it, but I agree that refactoring all the tests doesn't look great either.

Reply:
If I were to read that test in isolation, I would interpret that as explicitly making sure the table is empty before querying. 

The reason I'd be hesitant to refactor all of the tests within this class would be that they follow the same general pattern as the tests for Paging2 as well as the upcoming `KeyedPagingSource`. Jumping between related files would make this one stick out. 

Regardless, this isn't a huge deal. Thank you for catching this bug and patching it :)

Reply:
Ahhhh fair, I hadn't looked at the tests for Paging2. I guess if there should be a refactoring, then it should be done for both Paging2 and Paging3, but I think such a refactoring can be put on hold until it's really necessary.

Reply:
Fixed in https://github.com/cashapp/sqldelight/pull/2219/commits/5d2fbf42289d591b35e0fa78cfc35423751611f6

Reply:
Thanks for the change! @AlecStrong LGTM